mitodl/mit-open
# Base Api wrapper and themes Apis

#### What are the relevant tickets?
Fixes #2 

#### What's this PR do?
Adds APIs around themes

#### How should this be manually tested?
Set values for the `MIT_OPEN_REDDIT_*` env variables in `.env`. the anonymous ones should be from a script-based app and the authenticated one should be for a web based app.

```python
from themes.api import Api()

api = Api()
api.list_themes().next()
api.user = 'fake'  # hacky way to fake a user
api.create_theme('test1', 'Test 1')
```

Verify you see your new theme in your local reddit.
